docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -0,0 +1,196 @@
+                        .url("http://serviceId/hello").build();
+Response response = builder.build().newCall(request).execute();
+----
+
+== Retrofit with OkHttpClient and Spring Cloud LoadBalancer
+
+First, add the `spring-cloud-square-retrofit`  and `spring-cloud-square-okhttp` dependencies to your project:
+
+[source,xml]
+----
+<dependencies>
+    <dependency>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-square-retrofit</artifactId>
+		<version>0.4.0-MILESTONE</version>
+    </dependency>
+    <dependency>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-square-okhttp</artifactId>
+		<version>0.4.0-MILESTONE</version>
+    </dependency>
+</dependencies>
+----
+
+Use the `@EnableRetrofitClients` annotation to allow us automatically instantiate and inject Retrofit clients for you, and create a `@LoadBalanced`-annotated `OkHttpClient.Builder` bean to be used under the hood:
+
+[source,java]
+----
+@Configuration
+@EnableRetrofitClients
+class OkHttpClientConfig {
+
+@Bean
+@LoadBalanced
+public OkHttpClient.Builder okHttpClientBuilder() {
+    return new OkHttpClient.Builder();
+    }
+}
+----
+
+Create a Retrofit client and annotate it with `@RetrofitClient`, passing the `serviceId` of your service as argument (the annotation can also be used to pass a custom configuration, containing user-crated interceptors for the Retrofit client):
+
+[source,java]
+----
+@RetrofitClient("serviceId")
+interface HelloClient {
+	@GET("/")
+	Call<String> hello();
+}
+----
+
+Make sure to use Retrofit method annotations, such as `@GET("/")`.
+You can now inject the Retrofit client and use it to run load-balanced calls (using `serviceId` instead of actual `host:port`):
+
+[source,java]
+----
+class AService {
+
+    @Autowired
+    HelloClient client;
+
+	public String hello() throws IOException {
+		return client.hello().execute().body();
+	}
+}
+----
+
+A full sample for load-balanced-OkHttpClient-based Retrofit clients can be found https://github.com/spring-cloud-samples/spring-cloud-square-retrofit-web[here].
+
+== Retrofit with WebClient and Spring Cloud LoadBalancer
+
+First, add the `spring-cloud-square-retrofit` and `spring-boot-starter-webflux` starter dependencies to your project:
+
+[source,xml]
+----
+<dependencies>
+    <dependency>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-square-retrofit</artifactId>
+		<version>0.4.0-MILESTONE</version>
+    </dependency>
+    <dependency>
+    	<groupId>org.springframework.boot</groupId>
+    	<artifactId>spring-boot-starter-webflux</artifactId>
+    </dependency>
+</dependencies>
+----
+
+Use the `@EnableRetrofitClients` annotation to allow us automatically instantiate and inject Retrofit clients for you, and create a https://docs.spring.io/spring-cloud-commons/docs/current/reference/html/#webclinet-loadbalancer-client[`@LoadBalanced`-annotated `WebClient.Builder` bean] to be used under the hood:
```

Comment:
Change to:

Use the `@EnableRetrofitClients` annotation to let us automatically instantiate and inject Retrofit clients for you. Then create a https://docs.spring.io/spring-cloud-commons/docs/current/reference/html/#webclinet-loadbalancer-client[`@LoadBalanced`-annotated `WebClient.Builder` bean] to be used under the hood: